glustercli/cmd/volume-profile.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Profile Info Flags
+	flagProfileInfoPeek            bool
+	flagProfileInfoIncremental     bool
+	flagProfileInfoIncrementalPeek bool
+	flagProfileInfoCumulative      bool
+	flagProfileInfoClear           bool
+)
+
+var volumeProfileCmd = &cobra.Command{
+	Use:   "profile",
+	Short: "Gluster volume profile",
```

Comment:
can you add some more info about what ```profile command``` does in  ```Long``` command

Reply:
Done